utilities/pipelines/e2eValidation/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -156,6 +156,7 @@ function Invoke-ResourceRemoval {
 
             $resourceGroupName = $ResourceId.Split('/')[4]
             $resourceName = Split-Path $ResourceId -Leaf
+            $subscriptionId = $ResourceId.Split('/')[2]
```

Comment:
Please add a couple of pipeline badges

Reply:
Just added a badge. However like mentioned in the PRs description none of the pipeline badges could actually touch on the code that was changed

Reply:
Passed